### DISCUSSION TITLE
Refresh all devices?

### DISCUSSION THREAD

**Discussion by hotnuts21** [make-all/tuya-local#2693] [Q&A]
It's highly likely I'm missing something obvious, but is there a way to refresh all the devices settings that I have set up? 

I'm noticing that more and more devices stop working in HA, they remain working and connected in the Smart app and in my Google home app. But in HA some items are unavailable and I can't really work out why. But once they get to this state, I get the 'unable to connect to your device with those settings ...' message in the device settings. 

So I have to manually delete and re-add each device. Which is a pain at the mo as I have to do it all through the app and trying to scan a qr code with the same device is becoming frustrating (I'm printing it out 😂) 

Anyway if there was an option to connect to smart app, then refresh all devices currently added, would be handy.  Sometimes I miss one without realising it's not active. 

It's mainly bulbs and switches I have an issue with ans currently 40% of my devices seem to not be working.

**Comment by pilijic2** [comment]
Sorry, but I have to ask... do you have your equipment in a static DHCP table or do they still maintain dynamic addresses? .. use custom YAML? Personally, I have many models of equipment and I cannot replicate the error you describe. The only thing I can think of is that the IP in the DHCP or the token key has changed when you removed your equipment from the network and re-incorporated it into the app.

**Comment by make-all** [comment]
In Settings / Devices / Tuya Local / <device> Configure, you can edit all the settings it makes sense to reconfigure.
